src/state-mutation.js

```diff
@@ -93,7 +93,12 @@ export const removeEntityFromState = (state, entity) => {
       curVal => curVal.filter(l => l.get('id') !== entity.get('id'))
     );
 
-    entity.get('relationships').forEach(relationship => {
```

Comment:
After implementing the fix in `apiRequest` I noticed that this error was popping up when trying out the build locally

In `updateOrInsertEntity` I noticed the exact same early return and figured we might want the same logic in case the relationship was `undefined` like I was seeing in my test run